File: C++/344. Reverse String.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int i;
        char c;
        for(i=0;i<s.size()/2;i++){
            c=s[i];
            s[i]=s[s.size()-1-i];
            s[s.size()-1-i]=c;
        }
    }
};

File: JAVA/344. Reverse String.java
class Solution {
    public void reverseString(char[] s) {
        int i;
        char c;
        for(i=0;i<s.length/2;i++){
            c=s[i];
            s[i]=s[s.length-1-i];
            s[s.length-1-i]=c;
        }
    }
}